pkg/discovery/worker/k8s_discovery_worker.go

```diff
@@ -158,9 +169,10 @@ func (worker *k8sDiscoveryWorker) executeTask(currTask *task.Task) *task.TaskRes
 				defer wg.Done()
 
 				w.ReceiveTask(resourceMonitorTask)
-				t := time.NewTimer(timeout)
+				t := time.NewTimer(timeoutSecond)
 				go func() {
-					glog.V(2).Infof("A %s monitoring worker is invoked.", w.GetMonitoringSource())
+					glog.V(2).Infof("A %s monitoring worker from discovery worker %v is invoked.",
+						w.GetMonitoringSource(), worker.id)
```

Comment:
Would it be helpful to add `resourceMonitorTask` to the log message so that we'll be able to know which node this worker is about to process?
Something like:
```
glog.V(2).Infof("A %s monitoring worker from discovery worker %v is invoked for task %s.",
 w.GetMonitoringSource(), worker.id, resourceMonitorTask)
```